perfkitbenchmarker/centurylink/util.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utilities for working with Centurylink Cloud resources."""
+
+import os
+import random
+import string
+
+
+def AutoGenerateServerName():
+    """Returns an auto-generated server name"""
+
+    randAlfaKey = ''
+
+    for i in range(0, 3):
+        randAlfaKey += random.choice(string.ascii_uppercase)
+
+    randIntKey = random.randint(100, 999)
+    serverName = '%s%s' % (randAlfaKey, randIntKey)
+    return serverName
+
+
+def ParseCLIResponse(output):
```

Comment:
The CLI doesn't have an output format option does it? A lot of other CLI's have the option to output json and that makes parsing them a lot easier.